_data/icons/etherlite.json

```diff
@@ -0,0 +1,8 @@
+[
+    {
+      "url": "ipfs://QmbNAai1KnBnw4SPQKgrf6vBddifPCQTg2PePry1bmmZYy",
+      "width": 80,
+      "height": 80,
```

Comment:
```suggestion
      "height": 88,
```

Reply:
Updated the requested dimensions for logo.